### PR TITLE
Fix docstring line length

### PR DESCRIPTION
**Pull request by cedricconol**
This is a follow through on #136. 

Fixed docstring line length of `preprocessing`, `representation`, and `visualization` modules to respect numpy docstring guide of 75 characters max length. `TODO`s in docstrings were left untouched.

### CODE REVIEW

File: texthero/representation.py [hunk 1]
```diff
@@ -782,8 +802,12 @@ def dbscan(
     --------
     >>> import texthero as hero
     >>> import pandas as pd
-    >>> s = pd.Series(["Football, Sports, Soccer", "music, violin, orchestra", "football, fun, sports", "music, enjoy, guitar"])
-    >>> s = s.pipe(hero.clean).pipe(hero.tokenize).pipe(hero.tfidf).pipe(hero.flatten) # TODO: when others get Representation Support: remove flatten
+    >>> s = pd.Series(["Football, Sports, Soccer", "music, violin, orchestra",
+    ...                "football, fun, sports", "music, enjoy, guitar"])
+    >>> s = s.pipe(hero.clean) \
```
Comment (jbesomi) [inline texthero/representation.py:807]: Good catch!
````
s = (
   s.pipe(hero.clean)
   ...
    .pipe(hero.flatten)
)
```

might be even nicer? this is how we do it for instance in the README ...

File: texthero/visualization.py [hunk 2]
```diff
@@ -61,11 +64,16 @@ def scatterplot(
...
     >>> import pandas as pd
-    >>> df = pd.DataFrame(["Football, Sports, Soccer", "music, violin, orchestra", "football, fun, sports", "music, fun, guitar"], columns=["texts"])
+    >>> df = pd.DataFrame(["Football, Sports, Soccer",
+    ...                    "music, violin, orchestra", "football, fun, sports",
+    ...                    "music, fun, guitar"], columns=["texts"])
     >>> df["texts"] = hero.clean(df["texts"]).pipe(hero.tokenize)
-    >>> df["pca"] = hero.tfidf(df["texts"]).pipe(hero.flatten).pipe(hero.pca, n_components=3) # TODO: when others get Representation Support: remove flatten
-    >>> df["topics"] = hero.tfidf(df["texts"]).pipe(hero.flatten).pipe(hero.kmeans, n_clusters=2) # TODO: when others get Representation Support: remove flatten
-    >>> hero.scatterplot(df, col="pca", color="topics", hover_data=["texts"]) # doctest: +SKIP
+    >>> df["pca"] = hero.tfidf(df["texts"]).pipe(hero.flatten) \
```
Comment (jbesomi) [inline texthero/visualization.py:71]: same comment as before

File: texthero/representation.py [hunk 3]
```diff
@@ -685,8 +701,12 @@ def kmeans(
     --------
     >>> import texthero as hero
     >>> import pandas as pd
-    >>> s = pd.Series(["Football, Sports, Soccer", "music, violin, orchestra", "football, fun, sports", "music, fun, guitar"])
-    >>> s = s.pipe(hero.clean).pipe(hero.tokenize).pipe(hero.term_frequency).pipe(hero.flatten) # TODO: when others get Representation Support: remove flatten
+    >>> s = pd.Series(["Football, Sports, Soccer", "music, violin, orchestra",
+    ...                "football, fun, sports", "music, fun, guitar"])
+    >>> s = s.pipe(hero.clean) \
```
Comment (jbesomi) [inline texthero/representation.py:706]: same comment as before/after

**Comment by jbesomi** [comment]
That's soo cool Cedric! 🎉 , thank you very much!

I just added a comment regarding a best-practice we might want to follow (also, we should add this same best-practice under CONTRIBUTING for informing new contributors, would you like to do it?)

**Comment by cedricconol** [comment]
Hi, Jonathan. Done updating the docstrings to follow best practice.

**Comment by cedricconol** [comment]
> That's soo cool Cedric!  , thank you very much!
> 
> I just added a comment regarding a best-practice we might want to follow (also, we should add this same best-practice under CONTRIBUTING for informing new contributors, would you like to do it?)

Yup, I'd like to do it! So far these are the things that I want to add:
1. Manually limit docstring line length to 75 characters due to blacks's limitation
2. Multiline code best practice as discussed above

I'll open a PR for this and maybe you can add more.

**Comment by jbesomi** [comment]
Amazing, thank you!! 🎉 🎉

**Pull request merged**